docs/mkdocs.yml

```diff
@@ -22,7 +22,7 @@ plugins:
 
 markdown_extensions:
     -   toc:
-            permalink: true
+            permalink: "#"
```

Comment:
To make it persistent with auto-generated links: `page#header`
It's not clear that `¶` is a link.

permalink: true
![Screenshot from 2021-03-02 15-50-28](https://user-images.githubusercontent.com/1104540/109650992-0c20e200-7b6f-11eb-8594-0d0c4ed990ea.png)

permalink: "#"
![Screenshot from 2021-03-02 15-49-56](https://user-images.githubusercontent.com/1104540/109650991-0b884b80-7b6f-11eb-851c-910082ccbe88.png)